src/Components/SampleVisualizer/SampleVisualizer.tsx

```diff
@@ -79,6 +81,9 @@ export const SampleVisualizer: FunctionComponent<SampleVisualizerProps> = ({ iTw
           }
 
           setAppReady(true);
+
+          // Fix to add Hilite ruleset until Bug #599922 is addressed
+          await Presentation.presentation.rulesets().add((HILITE_RULESET as any).default as Ruleset);
```

Comment:
It may be a good idea to move this before the setAppReady(true). It may run into a race condition with rendering the sample otherwise

Reply:
Ok. I'll also add it in the `!iTwinViewerReady` if block since iTwinViewer samples will get this through the hook